app/src/main/java/com/semina/semi_na/data/db/entity/Semina.java

```diff
@@ -156,4 +160,17 @@ public MajorCategory getMajorCategory() {
     public HobbyCategory getHobbyCategory() {
         return hobbyCategory;
     }
+
+    public boolean isClosed() {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy년 MM월 dd일", Locale.KOREA);
+        try {
+            Date seminarDate = dateFormat.parse(date);
+            Date currentDate = new Date();
+
+            return seminarDate != null && seminarDate.before(currentDate);
+        } catch (ParseException e) {
+            e.printStackTrace();
+            return false;
```

Comment:
요거 굳이 try catch문으로 할 필요가 있을까?
파이어스토어에 세미나 생성될 때 `date` 필드는 무조건 있을 거 같은데
```suggestion
  Date seminarDate = dateFormat.parse(date);
  Date currentDate = new Date();
  
  return SeminarDate.before(currentDate);
```
요렇게 바꾸는게 가독성은 더 좋을 것 같아

Reply:
오 넵!!